youtube/generic.md

```diff
@@ -1,37 +1,46 @@
-...
-
-I now have a 2nd YouTube channel for short video clips of 60s or less, subscribe ... 
-https://www.youtube.com/eddiejaoudetv
-
-JOIN our inclusive open source community EddieHub now for FREE ... http://eddiehub.org, also checkout my website and sign up to our community NEWSLETTER... http://eddiejaoude.io for my live stream + video schedule and community calls
-
-Also join our community GitHub organisation by creating an issue with the type "invite me to the organisation" :)
-
----------------------------------------------------------------------------------------------------
-
-A BIG thank you to my GitHub Sponsors:
-***
-Stephen Mount http://github.com/stemount
-Nicholas Carrigan https://github.com/nhcarrigan
-***
-
-- Santosh Yadav https://github.com/santoshyadavdev
-- Andrew Cunliffe https://github.com/andrew-cunliffe
-- Alexander Stoichkov https://github.com/SashoStoichkov
-- Allan Regush https://github.com/AllanRegush
-
----------------------------------------------------------------------------------------------------
-
-Don’t forget to support my channel by subscribing below, it’s free, and also share with your friends. Subscribe now!!
-https://www.youtube.com/eddiejaoude?sub_confirmation=1
-
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
-
-http://eddiejaoude.io 
-https://github.com/eddiejaoude
-http://youtube.com/eddiejaoude
-http://twitter.com/eddiejaoude
-http://instagram.com/eddiejaoude
-http://linkedin.com/in/eddiejaoude
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+## YouTube Channel
+Don’t forget to support my channel by subscribing it! It’s free: [youtube.com/eddiejaoude](https://youtube.com/eddiejaoude?sub_confirmation=1)
+**Make sure to tell your friends about it, so that they also learn something new!**
+
+## YouTube Shorts
+I have another YouTube channel for short clips, please **take a moment subscribing it: [youtube.com/eddiejaoudetv](https://youtube.com/eddiejaoudetv?sub_confirmation=1)**
+
+## Join Our Community
+Join our inclusive open source community EddieHub now for **free** at [eddiehub.org](https://www.eddiehub.org)
+ 
+---
+
+Make sure to checkout my website and sign up to our community **newsletter**: [eddiejaoude.io](https://www.eddiejaoude.io) for my live stream + video schedule and community calls
+
+Join Our **Community GitHub Organization**:
+ 
+- Go to [EddieHubCommunity/support/issues/new/choose](https://github.com/EddieHubCommunity/support/issues/new/choose) 
```

Comment:
These unordered lists don't render properly when copied and pasted. Maybe we can edit and make it look like-
```suggestion
\- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
\- Choose The Issue Type Of **Invite Me To The Organization**
```
this renders- 
\- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
\- Choose The Issue Type Of **Invite Me To The Organization**